sotodlib/site_pipeline/update_obsdb.py

```diff
@@ -235,7 +235,12 @@ def main(config: str,
             if stream_ids is not None:
                 bookcartobsdb.add_obs_columns(["wafer_count int"])
                 very_clean["wafer_count"] = len(stream_ids)
-
+                bookcartobsdb.add_obs_columns(["ufms str"])
+                very_clean["ufms"] = (", ".join(stream_ids)).replace("ufm_","")
+            wafer_slots = index.pop("wafer_slots")
+            if wafer_slots is not None:
+                bookcartobsdb.add_obs_columns(["wafer_slots str"])
+                very_clean["wafer_slots"] = (', '.join([sl["wafer_slot"] for sl in wafer_slots]))
```

Comment:
Request for simply  ','.join rather than ', '.  That's gentler on users who are going to split this string, or who rely on whitespace being used to separate true columns in a text output...